### PR TITLE
some clarifying comments

### PR DESCRIPTION
**Pull request by hnvakil**

### CODE REVIEW

File: src/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,16 @@
 # syntax=docker/dockerfile:1
 FROM python:3.7-alpine
+# may want to swap out this for a different image? causes problems with matplotlib
```
Comment (berwinlan) [inline src/Dockerfile:3]: to be nitpicky, can you add some line breaks so it's clear which comments go with which lines? or put the comments in the same line

File: src/Dockerfile [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 WORKDIR /code
 ENV FLASK_APP=app.py
 ENV FLASK_RUN_HOST=0.0.0.0
+# this may be bad for long term security but I don't understand enough about
+# access points to know for sure
 RUN apk add --no-cache gcc musl-dev linux-headers
+# what does this line do? copied from the initial guide followed but not sure
```
Comment (berwinlan) [inline src/Dockerfile:12]: lmao nice

File: src/docker-compose.yml [hunk 3]
```diff
@@ -2,7 +2,11 @@ version: "3.9"
 services:
   web:
     build: .
+    # this builds the current folder so you have to run docker compose at the
+    # level containing this yml
     ports:
       - "8000:3000"
+    # why is this 3000? not sure but it works
```
Comment (berwinlan) [inline src/docker-compose.yml]: put comments at same indentation level as the thing being commented on 👉👈

**Review by berwinlan — COMMENTED** [review]
See comments

**Comment by hnvakil** [comment]
Hey @berwinlan, I made some changes based on your feedback. Mind taking another look?

**Review by berwinlan — CHANGES_REQUESTED** [review]
put your comments above bestie!!!

**Comment by hnvakil** [comment]
> 

comments
bestie

**Comment by hnvakil** [comment]
Hey bestie berwin, made the swap. Let me know if there's anything else :)

**Review by berwinlan — APPROVED** [review]
Looks great!

**Pull request merged**